### ISSUE TITLE
MPV icon redesign

### ISSUE THREAD

**Issue by daffc32** [numixproject/numix-core#4784]
Current icon is not on the Numix level of greatness. I know that the original MPV icon is not a piece of art either but current icon is generic and bland. If you just make a flat version of original MPV icon it would be a way better.

**Comment by JazzJ** [comment]
I agree with @daffc32 it's definitely time for MPV icon redesign. This is my humble sketch how I think new icon should look. it's amateur work, I'm not a graphic designer sadly, hope someone can work on this idea if people find this worth working on.

![2019-02-16_193033](https://user-images.githubusercontent.com/47699008/52903908-e6913c00-3224-11e9-99d9-f4c3f6f10762.jpg)

**Comment by Foggalong** [comment]
@JazzJ I appreciate the effort, but that's likely not a design we'd go with. It's not matching the [upstream branding](http://tipsonubuntu.com/wp-content/uploads/2017/02/mpv-icon.jpg) which is part of the issue but nor does the symbol idea fit well with our [design guidelines](https://github.com/numixproject/numix-core/wiki/Guidelines).

**Comment by palob** [comment]
Looking at the original icon I still see just a triangle. On top of a circle and a two-tone background however.

**Comment by JazzJ** [comment]
@Foggalong Oh, I see. Would something like this be more acceptable?
![2019-02-17_131918](https://user-images.githubusercontent.com/47699008/52912789-babe9680-32b6-11e9-8ce5-4cc06da0133b.jpg)

**Comment by Foggalong** [comment]
@JazzJ It's closer, and whatever we do will likely look something like that symbolism wise, but the baseplate wouldn't be something we'd do guidelines wise.

**Comment by daffc32** [comment]
I like this second suggestion from @JazzJ very much. Simple, elegant, that's a Numix style. @Foggalong SMPlayer has the same icon as MPV. Maybe SMPlayer can use a different shade than MPV?
